### PR TITLE
CUDA 11.0 and 11.1 dockerfiles modified to build pytorch and tensorfl…

### PR DESCRIPTION
**Pull request by bsewp045**
…ow images

Closes https://github.com/StatCan/aaw-kubeflow-containers/issues/345 
# Description

**What your PR adds/fixes/removes**

# Tests / Quality Checks

## Are there breaking changes?
Ask yourself the next question;
- [ ] Do we want to maintain the previous image from which we had to do breaking changes from?

If no, then carry on. If yes, there is a breaking change and we **want to maintain the previous image** do the following
- [ ] Create a new branch for the current version (ex v1) based off the current master/main branch
- [ ]  Increment the tag in the CI for pushes to master/main (v1 to v2)
- [ ] Change the CI that on pushes to the newly created "v1" branch (the name of the newly created branch we want to maintain is) it will push to the ACR. 
## Automated Testing/build and deployment
- [ ] Does the image pass CI successfully (build, pass vulnerability scan, and pass automated test suite)?
- [ ] If new features are added (new image, new binary, etc), have new automated tests been added to cover these?
- [ ] If new features are added that require in-cluster testing (e.g. a new feature that needs to interact with kubernetes), have you added the `auto-deploy` tag to the PR before pushing in order to build and push the image to ACR so you can test it in cluster as a custom image?

## JupyterLab extensions

- [ ] Are all extensions "enabled" (`jupyter labextension list` from inside the notebook)?

## VS Code tests

- [ ] Does VS Code open?
- [ ] Can you install extensions?

## Code review

- [ ] Have you added the `auto-deploy` tag to your PR before your most recent push to this repo?  This causes CI to build the image and push to our ACR, letting reviewers access the built image without having to create it themselves
- [ ] Have you chosen a reviewer, attached them as a reviewer to this PR, and messaged them with the SHA-pinned image name for the final image to test on the **dev cluster** (e.g. `k8scc01covidacrdev.azurecr.io/jupyterlab-cpu:746d058e2f37e004da5ca483d121bfb9e0545f2b`)?

### CODE REVIEW

**Review by Jose-Matsuda — CHANGES_REQUESTED** [review]
This is just a preliminary review just wanted to jot down some notes.
So in regards to _upgrading_ `CUDA` is that something you'd want to tackle in a separate pr or can it be done here? 

Or would it be because Brendan did not respond in the `@`? 
If this is the case I think Brendan would be ok with progressing with an update, as long as it doesn't break anything (I see you mentioned `libnccl2, libcudnn8` as possible pain points, but if those can be upgraded as well to work with `CUDA 11.7` then I personally don't see an issue

**Comment by Jose-Matsuda** [comment]
I would also change your commit message title to be a little more like https://www.conventionalcommits.org/en/v1.0.0/ 

Similar to mine 
image

So in this case maybe something like 
`fix(tensorflow pytorch): update keys` and then in the description of the commit you can talk about it more kind of like
image
but it's up to you if you think the commit message needs more context than the title that really states what was changed

File: docker-bits/1_CUDA-11.0.Dockerfile [hunk 1]
```diff
@@ -9,16 +9,19 @@
...
-    echo "deb https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64 /" > /etc/apt/sources.list.d/nvidia-ml.list && \
+    curl -fsSL https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/3bf863cc.pub | apt-key add - && \
     apt-get purge --autoremove -y curl \
     && rm -rf /var/lib/apt/lists/*
 
 ENV CUDA_VERSION 11.0.3
 
 # For libraries in the cuda-compat-* package: https://docs.nvidia.com/cuda/eula/index.html#attachment-a
-RUN apt-get update && apt-get install -y --no-install-recommends \
+RUN apt-key del 7fa2af80 && \
```
[the remark this answers is not in the document]
Comment (bsewp045) [reply docker-bits/1_CUDA-11.0.Dockerfile:19]: I don't mind updating the CUDA version now

**Review by Jose-Matsuda — APPROVED** [review]
LGTM

**Pull request merged**